build/mbl/build.sh

```diff
@@ -947,8 +961,9 @@ while true; do
        # quoted because it will form multiple options rather than one
        # option. However ideally each option in the list should be
        # quoted.
+       printf "\nCalling: DISTRO=%s bitbake %s\n\n" "$distro" "$images"
        # shellcheck disable=SC2086
-       bitbake $images
+       DISTRO="$distro" bitbake $images
```

Comment:
Why are we setting `DISTRO` in the environment? Isn't it in auto.conf or local.conf or something?

Reply:
Good question. I added this at the beginning of the implementation where the bitbake build directory was the same for different distros. Because now we have a build dir per distro this is not needed. I'll remove it.

Reply:
removed at e8b6ef4.